pkg/chains/formats/intotoite6/intotoite6_test.go

```diff
@@ -62,13 +62,13 @@ func TestCreatePayload1(t *testing.T) {
 				BuildFinishedOn: &e1BuildFinished,
 			},
 			Materials: []slsa.ProvenanceMaterial{
-				{URI: "git+https://git.test.com.git", Digest: slsa.DigestSet{"sha1": "abcd"}},
+				{URI: "git+https://git1.test.com.git", Digest: slsa.DigestSet{"sha1": "sha:taskrun"}},
 			},
 			Invocation: slsa.ProvenanceInvocation{
 				Parameters: map[string]v1beta1.ArrayOrString{
 					"IMAGE":             {Type: "string", StringVal: "test.io/test/image"},
-					"CHAINS-GIT_COMMIT": {Type: "string", StringVal: "abcd"},
```

Comment:
It would be great to improve the test coverage to ensure the default value is indeed used if there are results and no explicit parameters.

Reply:
> ensure the default value is indeed used if there are results and no explicit parameters.

Sorry do you mean if there are **NO** results and no explicit params? If so, that makes sense to me. I just added the case to `taskrun2.json`. Please take a look.
Thanks